SDWebImage/Core/SDAnimatedImagePlayer.h

```diff
@@ -79,7 +79,7 @@ typedef NS_ENUM(NSUInteger, SDAnimatedImagePlaybackMode) {
 /// The provider can be any protocol implementation, like `SDAnimatedImage` or `SDImageGIFCoder`, etc.
 /// @note This provider can represent mutable content, like progressive animated loading. But you need to update the frame count by yourself
 /// @param provider The animated provider
-+ (nullable instancetype)playerWithProvider:(nonnull id<SDAnimatedImageProvider>)provider;
++ (nullable instancetype)playerWithProvider:(nonnull id<SDAnimatedImageProvider>)provider NS_SWIFT_NAME(init(player:));
```

Comment:
Here a player is created with `SDAnimatedImageProvider` as a parameter.
So probably the correct Swift name would be `NS_SWIFT_NAME(init(provider:))`

Reply:
It's a typo, I'll fix it this weekend.